### PR TITLE
initially hide node conditions

### PR DESCRIPTION
**Pull request by mrIncompetent**
This PR adds a query parameter called `hideInitialConditions` which hides node conditions for nodes which are younger than 5 minutes.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/node_v2.go [hunk 1]
```diff
@@ -104,9 +104,11 @@ func apiNodeStatus(status apiv2.NodeStatus, inputNode *corev1.Node) apiv2.NodeSt
 		})
 	}
 
-	reason, message := parseNodeConditions(inputNode)
-	status.ErrorReason += reason
-	status.ErrorMessage += message
+	if time.Now().Sub(inputNode.CreationTimestamp.Time).Minutes() > 5 {
```
Comment (metalmatze) [inline api/pkg/handler/node_v2.go]: I'd go for: `time.Since(inputNode.CreationTimestamp.Time).Minutes()`

**Review by metalmatze — CHANGES_REQUESTED** [review]
I like the approach. :joy: 
Only one small comment.

File: api/pkg/handler/node_v2.go (hunk 1 above)
Comment (mrIncompetent) [reply api/pkg/handler/node_v2.go]: linting complaint was exactly that. Fixed

Comment (metalmatze) [reply api/pkg/handler/node_v2.go]: No need for a linter, when you have me as a reviewer. :joy:

File: api/pkg/handler/node_v2.go [hunk 2]
```diff
@@ -104,9 +106,11 @@ func apiNodeStatus(status apiv2.NodeStatus, inputNode *corev1.Node) apiv2.NodeSt
 		})
 	}
 
-	reason, message := parseNodeConditions(inputNode)
-	status.ErrorReason += reason
-	status.ErrorMessage += message
+	if time.Since(inputNode.CreationTimestamp.Time).Minutes() > initialConditionParsingDelay {
```
Comment (alvaroaleman) [inline api/pkg/handler/node_v2.go]: MMh. I really dislike time-based approaches and I really dislike that is not possible at all anymore to get the info sooner.

What about adding an optional url parameter that when set will circumvent this?

**Comment by mrIncompetent** [comment]
PTAL @metalmatze @alvaroaleman

**Comment by mrIncompetent** [comment]
@alvaroaleman @metalmatze PTAL

**Review by metalmatze — APPROVED** [review]

**Comment by alvaroaleman** [comment]
i like :+1:

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**